model/schema.json

```diff
@@ -21,7 +21,7 @@
         "version"
       ]
     },
-    "host": {
+    "host-metadata": {
```

Comment:
Any reason you renamed this? Before it matched ECS so nicely :-)

Reply:
Think of this name as just a variable. I updated it so it matched `agent-metadata`, and `server-metadata`. But these are just variables that are referenced by other objects in the schema.

So `host-metadata` is used like:

```
"host": { "$ref": "#/definitions/host-metadata" }
```

So the resulting document in ES will be:

```
host: {
  ...fields from host-metadata...
}
```

Reply:
Got it.